src/features/iam/iamHelpers.ts

```diff
@@ -31,7 +32,38 @@ async function createDefaultProject(): Promise<void> {
   ext.explorerTree.refresh()
 }
 
+const renameProject = async ({ projectId }: { projectId: string }) => {
+  const project = await iamState.getProjectById(projectId)
+  if (!project) return
+  const name = await window.showInputBox({
+    prompt: `Enter the new name for the project "${project.name}":`,
+  })
+  if (!name) return
+
+  try {
+    const consoleAuthToken = await authHelper.getConsoleAuthToken()
+    const data = await window.withProgress(
+      {
+        location: ProgressLocation.Notification,
+      },
```

Comment:
Did you accidentally remove notification title?

Reply:
I had a brief chat with Carlos about it, the message itself flashes up and ends so quickly that he suggested that it was good not to over to complicate it and remove it.

Reply:
It depends on the connection quality tho. But I'm okay with this